saga/src/worker.ts

```diff
@@ -0,0 +1,81 @@
+import { InjectedSinks, NativeConnection, Worker, WorkerOptions } from '@temporalio/worker';
+import * as fs from 'fs';
+import { createActivities } from './activities';
+import { createClients } from './clients';
+import { Env, getEnv, isRemoteEnv } from './env';
+import { LoggerSinks } from './workflows';
+
+// worker
+async function run(env: Env) {
+  const sinks: InjectedSinks<LoggerSinks> = {
+    logger: {
+      info: {
+        fn(workflowInfo, message) {
+          console.log('workflow: ', workflowInfo.runId, 'message: ', message);
+        },
+        callDuringReplay: false, // The default
+      },
+      err: {
+        fn(workflowInfo, message) {
+          console.error('workflow: ', workflowInfo.runId, 'message: ', message);
+        },
+        callDuringReplay: false, // The default
+      },
+    },
+  };
+
+  // registrations
+  const singletonClients = await createClients();
+  const activities = createActivities(singletonClients) as any;
```

Comment:
Curious why you had to cast to any

Reply:
It'd be better to avoid that

Reply:
I think `proxyActivities` was choking on the explicit type being returned

Reply:
This should be related to `proxyActivities`, please remove and report if something isn't working as expected.

Reply:
Previously I had returned a interface type from `createActivities` but that was removed. I have dropped it here now.